auth-web/src/routes/router.ts

```diff
@@ -232,7 +232,7 @@ export function getRoutes (): RouteConfig[] {
       name: 'account',
       beforeEnter: (to, from, next) => {
         if (LaunchDarklyService.getFlag(LDFlags.EnableBusinessRegistryDashboard) && !to.query.noRedirect) {
-          window.location.href = ConfigHelper.getNewBusinessRegistryDashboardUrl()
+          window.location.href = ConfigHelper.getNewBusinessRegistryDashboardUrl() + '/' + to.params.orgId
```

Comment:
What about routing to the BRD by returning a route object instead? That way other params are passed through.

Reply:
What you've implemented looks fine, but after reading up on Nuxt Routing I thought you could return a route object instead of overriding the current window location. Thoughts?

Reply:
OK, we can revise this later (when we rip out this conditional and always redirect to the new BRD).

Resolving.